ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c

```diff
@@ -3306,11 +3351,19 @@ dbmdb_add_import_index(ImportCtx_t *ctx, const char *name, IndexInfo *ii)
             slapi_log_err(SLAPI_LOG_INFO, "dbmdb_db2index",
                           "%s: Indexing %s\n", job->inst->inst_name, mii->name);
         } else {
-            if (job->task) {
-                slapi_task_log_notice(job->task, "%s: Indexing attribute: %s", job->inst->inst_name, mii->name);
+            if (ii->ai->ai_indexmask == INDEX_VLV) {
+                if (job->task) {
+                    slapi_task_log_notice(job->task, "%s: Indexing VLV: %s", job->inst->inst_name, mii->name);
```

Comment:
Out of curiosity, where `mii` is freed? I see one `if` - 

```
if (a->offset_dbi) {
        *(MdbIndexInfo_t**)(((char*)ctx)+a->offset_dbi) = mii;
    }
```

But if we are not `a->offset_dbi` - what then?

Reply:
mii is stored in an avl so I think it is freed by: avl_free(ctx->indexes, (IFP) free_ii);

Reply:
But it's put there only in case `if (a->offset_dbi) {`, right?

Anyway, I probably miss something, as I'm not sure I fully understand what really happens here with `ii`, `ctx->job` and how we build the index list in general. I guess I need dive deeper here :)

Reply:
No: it is always stored in ctx->indexes map (last line of the function is: `avl_insert(&ctx->indexes, mii, cmp_mii, NULL);` )